source/core/parameters_lagrangian.cc

```diff
@@ -416,6 +416,11 @@ namespace Parameters
                           Patterns::Integer(),
                           "Checking frequency for dynamic load-balancing");
 
+        prm.declare_entry(
+          "load balance particle weight",
+          "2000",
```

Comment:
My research shows 15k and 50k show the best performance on the DEM side. Since both CFD-DEM and DEM use the same parameter, I think 2k is a decent default value.

Reply:
Okay, so since this value is common between cfd-dem and dem and since it is defined in the model parameters, what should I use as a default value?